default.nix

```diff
@@ -247,6 +247,8 @@ let
     maxLayers ? 1,
     # Deprecated: will be removed on v1
     contents ? null,
+    # Author, comment, command
+    metadata ? { created-by = "nix2container"; },
```

Comment:
Even if i would prefer to rename `metadata` to `history` in the Go code (which is internal code not exposed to end users), i think we could keep `metadata` here since `history` could be confusing for people who are not confident with the OCI spec.